### PR TITLE
Show total send count instead of remaining send count

### PR DESCRIPTION
**Pull request by hiemanshu**
This addresses issue #433.

### CODE REVIEW

File: src/components/AssignmentTexter.jsx [hunk 1]
```diff
@@ -125,8 +125,14 @@ class AssignmentTexter extends React.Component {
...
-    const currentIndex = this.state.currentContactIndex + 1
-    let ofHowMany = this.contactCount()
+    const { allContacts } = this.props
+    const allContactsCount = allContacts.length
+    const remainingContacts = this.contactCount()
+    const messagedContacts = allContactsCount - remainingContacts
+
+    const currentIndex = this.state.currentContactIndex + 1 + messagedContacts
+    let ofHowMany = allContactsCount
+    console.log(this.props)
```
Comment (shakalee14) [inline src/components/AssignmentTexter.jsx]: can we remove this log statement?

File: src/containers/TexterTodo.jsx [hunk 2]
```diff
@@ -39,14 +39,19 @@ class TexterTodo extends React.Component {
...
-    return (<AssignmentTexter
-      assignment={assignment}
-      contacts={contacts}
-      assignContactsIfNeeded={this.assignContactsIfNeeded.bind(this)}
-      refreshData={this.refreshData.bind(this)}
-      onRefreshAssignmentContacts={this.refreshAssignmentContacts}
-      organizationId={this.props.params.organizationId}
-    />)
+    const allContacts = assignment.allContacts
+    console.log(contacts)
```
Comment (shakalee14) [inline src/containers/TexterTodo.jsx]: and this log statement?

**Review by shakalee14 — APPROVED** [review]

**Comment by shakalee14** [comment]
qa'd locally successfully

**Comment by hiemanshu** [comment]
Fixed. Looks like the plugin I have that shows me in my editor that I have a console.log here (and in the git diffs when I'm staging) didn't catch these on .jsx files :(

**Pull request merged**